doc/GIT.md

```diff
@@ -147,13 +144,8 @@ After being added to whitelist you can trigger buildjobs by saying:
 * jenkins build [multiconfig-gcc-stable](https://build.libelektra.org/job/elektra-multiconfig-gcc-stable/) please
 * jenkins build [multiconfig-gcc47-cmake-options](https://build.libelektra.org/job/elektra-multiconfig-gcc47-cmake-options/) please
 * jenkins build [source-package-test](https://build.libelektra.org/job/elektra-source-package-test/) please
-* jenkins build [stable](https://build.libelektra.org/job/elektra-mergerequests-stable/) please
-* jenkins build [unstable](https://build.libelektra.org/job/elektra-mergerequests-unstable/) please
 * jenkins build [homepage](https://build.libelektra.org/job/elektra-homepage/) please
-* jenkins build [gcc-configure-debian-stretch-minimal](https://build.libelektra.org/job/elektra-gcc-configure-debian-stretch-minimal/) please
-* jenkins build [gcc-configure-debian-jessie-minimal](https://build.libelektra.org/job/elektra-gcc-configure-debian-jessie-minimal/) please
-* jenkins build [docker](https://build.libelektra.org/job/test-docker/) please
-* jenkins build [haskell](https://build.libelektra.org/job/elektra-haskell/) please
+* jenkins build [jenkinsfile](https://build.libelektra.org/jenkins/job/elektra-jenkinsfile/) please
```

Comment:
Is there no way to trigger individual parts of this job?

Reply:
No, I implemented it and reverted it as I did not see the point. Build all is fast enough. And if you need to debug an specific issue you can replicate the environment on your local workstation and can make use of incremental builds.

Reply:
> Job was aborted (restarts etc.)

if the job was aborted you need to rerun it completely anyway as you will not know if it could have completed the stages completely

> here was spurious error (some network connection failed, v2 segfault, ...)

but then you would not have a run where all results are published together, only the part that you reran.

> The problem can be easy fixed and/or local replication is unwanted, ...

if it can be easily fixed you want to do another build all anyway to see if it did not affect any of the other environments

> Jenkinsfile itself gets updates

that one i don't understand. if the jenkinsfile gets an update you will need to rebuild anything anyway as this is the only way to see what the jenkinsfile is actually doing?

> Always re-triggering everything is an easily avoidable waste of energy. It is imho already done too often.

What I see is the following behaviour: tests get run, some fail. new commit is uploaded, failing tests are rerun, either repeat or, rerun all afterwards. With the simplified setup you are only cutting out the middle part. And actually saving energy because you do the build all right away

if you run tests in an isolated matter then you don't need a ci system. you can just run your tests on each developer machine and be done with it. 

Reactivating the behavior is easy, I tested it and it works exactly as the current one. It is there and just commented out. However:
What will never be possible with this approach is having the current 40 build results all displayed individually like it is right now. one job means one result in the github pr ui (though you can modify the text displayed).

If you want to avoid waste of energy you need to provide clear guidlines on what you want to have tested and how because the current state is that there are tests rerun 10+ times in the too similar environments in different jobs. Which is the way bigger issue.

Reply:
> I already had this discussion with myself a week ago so I am happy to actually bring it to paper (or at list some hard drive in the cloud) :D.

Good that I asked then :smile: (Btw. I use github-backup)

> let me state some things that we completely left out of the discussion till now:

I think we are now on the same page, so there is no need to endlessly discuss what might be.
Lets face the fact:

It is a huge change, we both do not know the impact it will have on the community's behavior. So the only scientific way forward is to run an experiment if phrases for partial retrigger are needed or not.

> Hence my proposal moving forward: Keep build selection out for now.

How will we know if people find the partial retriggering useful if it is not available? It is easily possible that someone thinks he/she wants retriggering but after trying it out he/she finds it is useless. (E.g. because artifacts were not published.)

So I propose that we have it enabled and we will discuss in a future meeting if people found it useful. If there is doubt on its usefulness we remove it for simplicity reasons.